src/pages/UserList/UserList.jsx

```diff
@@ -0,0 +1,47 @@
+import { useSelector } from "react-redux";
+
+import { selectAllUsers } from "features/users/usersSlice";
+
+import { NoUsers } from "./NoUser/NoUsers";
+import { UserRecord } from "./UserRecord/UserRecord";
+
+import { thead, tbody, h2 } from "./UserList.module.css";
+import { Loader } from "components/Loader/Loader";
+
+export const UsersList = () => {
+  const users = useSelector(selectAllUsers) || [];
```

Comment:
rework selector to always return array instead of this check in each component
`|| []` should be in selector return value or this case should be eliinated at all (better)